internal/indexcoord/util_test.go

```diff
@@ -203,3 +203,14 @@ func Test_estimateIndexSizeByReq(t *testing.T) {
 		})
 	}
 }
+
+func Test_parseKey(t *testing.T) {
+	key := "test-ListObjects/1/"
```

Comment:
didn't test the case but we handled it in our code? why we handle it? If we handle in case we need to test on it
test-ListObjects/test/111